unit1/lesson19.rst

```diff
@@ -0,0 +1,20 @@
+At the end of angular class 2 students will be able to declare click handlers, take in user input and display it, and show lists of data using a loop directive.
+
+Angular Class 2 Objectives:
```

Comment:
Nice work on the Angular Class 2 objectives.  These are WAY better than the originals.